crates/turbopack-create-test-app/src/test_app_builder.rs

```diff
@@ -157,7 +166,14 @@ export default React.memo(Triangle);
 {b}
 {c}
 
-function Container({{ style }}) {{
+let EFFECT = () => {{}};
+
+/* @turbopack-bench:eval */
+/* @turbopack-bench:effect */
+
+function Component({{ style }}) {{
+    React.useEffect(EFFECT, [EFFECT]);
```

Comment:
This is be impossible to use for RSC, since they can't use `useEffect`.

Reply:
If we use the `Detector` component from https://github.com/vercel/turbo/pull/2698, instead of setting a top-level `EFFECT` binding, we'd set a `DETECTOR_PROPS` binding instead and render `<Detector {...DETECTOR_PROPS) />` within RSC components.

Reply:
So we need to have the detector in every file... I hope this doesn't cause much overhead.